var/spack/repos/builtin/packages/binutils/package.py

```diff
@@ -13,6 +13,8 @@ class Binutils(AutotoolsPackage, GNUMirrorPackage):
 
     maintainers = ['alalazo']
 
+    tags = ['core-packages']
```

Comment:
I would call this build-tools personally, this is where ld, ar and company are.  Could go either way I suppose.

Reply:
Done, now it has two tags